### PR TITLE
Feature/jwt authorization with db

### PR DESCRIPTION
**Pull request by DaniilAndCo**
I created Spring boot application with dependencies for working with MySql databases, Hibernate and Spring Security.
Connected project database and made Jwt authorization with the database, user table.

### CODE REVIEW

**Comment by DaniilAndCo** [comment]
The next my step is registration. I'll do it up to the next week and then we can have calling.

**Review by asenchuk — CHANGES_REQUESTED** [review]

**Comment by DaniilAndCo** [comment]
waiting your review on project structure

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.5.2</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.github.daniilandco</groupId>
+	<artifactId>spring-boot-initial</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>spring-boot-initial</name>
```
Comment (asenchuk) [inline pom.xml:14]: Should be the name of your project. `spring-boot-initial` is just a placeholder.

File: src/main/java/com/github/daniilandco/vehicle_sales_project/controller/request/RegisterRequest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    private Role role;
+
+    @Enumerated(value = EnumType.STRING)
+    private Status status;
+
+    public User getUser() {
+        return new User(
+                this.getFirstName(), this.getSecondName(),
+                this.getEmail(), this.getPhoneNumber(),
+                SecurityConfig.passwordEncoder().encode(this.getPassword()),
```
Comment (asenchuk) [inline src/main/java/com/github/daniilandco/vehicle_sales_project/controller/request/RegisterRequest.java:30]: ideally, this method should be moved to some kind of `UserService` where PasswordEncoder is injected and isn't statically referenced.

**Review by asenchuk — APPROVED** [review]
some changes are needed but let's move them to tickets

**Pull request merged**